.github/workflows/tweet-release.yml

```diff
@@ -0,0 +1,25 @@
+on:
+  release:
+    types: [released]
+
+jobs:
+  tweet:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: ethomson/send-tweet-action@v1
+        with:
+          status: >
+            We have a new ${{github.event.release.tag_name}} release of our
+            ${{ fromJson('{
+              "team-telnyx/telnyx-python": "Python",
+              "team-telnyx/telnyx-ruby": "Ruby",
+              "team-telnyx/telnyx-node": "Node",
+              "team-telnyx/telnyx-java": "Java",
+              "team-telnyx/telnyx-dotnet": "Dotnet",
+              "team-telnyx/telnyx-php": "PHP"
+            }')[github.event.repository.full_name] }} SDK!
+            Check out the details here: ${{ github.event.release.html_url }}
```

Comment:
clever!

Is there a way to centralize this workflow to team-telnyx so that we can update one place instead of living in each repo?

Reply:
There is an idea of an [organizational template](https://docs.github.com/en/actions/learn-github-actions/sharing-workflows-with-your-organization), but the end result is the same (`tweet-release.yml` checked into repo).